tempesta_fw/cache.c

```diff
@@ -523,14 +530,44 @@ tfw_cache_init(void)
 	return r;
 }
 
-void
-tfw_cache_exit(void)
+static void
+tfw_cache_stop(void)
 {
-	if (!tfw_cfg.cache)
+	if (!cache_cfg.cache)
 		return;
 
 	destroy_workqueue(cache_wq);
 	kmem_cache_destroy(c_cache);
 	kthread_stop(cache_mgr_thr);
-	tdb_close(db);
 }
+
+static TfwCfgSpec tfw_cache_cfg_specs[] = {
+	{
+		"cache", "off",
+		tfw_cfg_set_bool, &cache_cfg.cache
```

Comment:
Please don't place two statements at the same line - it makes the code harder to read